metadata.universal/resources/settings.xml

```diff
@@ -101,15 +101,15 @@
   <category label="30300">
     <setting type="lsep" label="30303"/>
     <setting type="sep"/>
-    <setting label="30301" type="bool" id="tmdbsearch" default="false" enable="!eq(2,true)"/>
-    <setting label="30308" type="select" values="bg|cs|da|de|el|en|es|fi|fr|he|hr|hu|it|ja|ko|nl|no|pl|pt|ru|sl|sv|tr|zh" id="tmdbsearchlanguage" default="en" visible="eq(-1,true)"/>
+    <setting label="30301" type="bool" id="tmdbsearch" default="false" enable="eq(3,false)"/>
```

Comment:
While i am changing this precise area right now to be user friendly, I believe the entire addon has gone way too far in being configurable.